src/components/Button/Button.js

```diff
@@ -1,9 +1,21 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import StyledButton from './StyledButton';
 
 function Button(props) {
   const content = props.label ? props.label : props.children;
   return <StyledButton {...props}>{content}</StyledButton>;
 }
 
+Button.defaultProps = {
+  type: 'button',
+  label: null,
+  children: null,
+};
+
+Button.propTypes = {
```

Comment:
Can you sort both the propTypes and the defaultProps?

Also can you add the following to the `.eslintrc` ?

`"react/sort-prop-types": 2`